recipes/kaitai_struct_cpp_stl_runtime/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class KaitaiStructCppStlRuntimeConan(ConanFile):
+    name = "kaitai_struct_cpp_stl_runtime"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://kaitai.io/"
+    description = "kaitai struct c++ runtime library"
+    topics = ("parsers", "streams", "dsl")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    options = {
+        "with_zlib": [True, False],
+        "with_iconv": [True, False]
+    }
+    default_options = {
+        "with_zlib": False,
+        "with_iconv": False
+    }
+    short_paths = True
+    _cmake = None
+
+    def requirements(self):
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.with_iconv:
+            self.requires("libiconv/1.16")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        if self.options.with_iconv:
+            self._cmake.definitions["STRING_ENCODING_TYPE"] = "ICONV"
+        else:
+            self._cmake.definitions["STRING_ENCODING_TYPE"] = "NONE"
```

Comment:
```suggestion
        self._cmake.definitions["STRING_ENCODING_TYPE"] = "ICONV" if self.options.with_iconv else "NONE"
```

Reply:
Should there be an option for zlib?

Reply:
Nope, there is no option for zlib. It's only find_package https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/blob/master/CMakeLists.txt#L35